### DISCUSSION TITLE
Is that possible to start mitmproxy with two  addon python script

### DISCUSSION THREAD

**Discussion by keepgoingandsmile** [mitmproxy/mitmproxy#6046] [Q&A] [answered]
Is that possible to start mitmproxy with two  addon python script? maybe like：
mitmdump -s ./anatomy2.py  ./anatomy2.py

**Comment by ZeroRin** [comment, marked as answer]
what is the point of adding it twice?
you may run addons from multiple python files with `mitmdump -s ./anatomy1.py -s ./anatomy2.py`, but you cannot run multiple addons with same name (even if they are actually from different files)
